elasticsearch/marvel.sls

```diff
@@ -0,0 +1,18 @@
+{% set marvel_version = salt['pillar.get']('elasticsearch:marvel:version', 'latest') %}
+
+install_marvel:
+  cmd:
+  - run
+  - name: '/usr/share/elasticsearch/bin/plugin -i elasticsearch/marvel/{{ marvel_version }}'
+  - require:
+    - pkg: elasticsearch
+  - unless: '/usr/share/elasticsearch/bin/plugin -l | grep marvel'
+
+
+restart_elasticsearch:
```

Comment:
Instead of restarting elasticsearch, just require the marvel state when you start it originally